### PR TITLE
Add pigz support to vzctl create

### PR DESCRIPTION
**Pull request by mrqwer88**
Add pigz support to vzctl create - https://bugs.openvz.org/browse/OVZ-5805

### CODE REVIEW

File: scripts/vps-create.in [hunk 1]
```diff
@@ -56,6 +56,8 @@ create_prvt()
 	case $HEADER in
 		' 1f 8b') # gzip
 			OPT=-z
+			# Use pigz for more fast uncompress template if we can
+			pigz -V 2>&1 && TAR_OPT="--use-compress-program=pigz ${TAR_OPT}" && OPT=""
```
Comment (kolyshkin) [inline scripts/vps-create.in]: It should be `pigz -V >/dev/null 2>&1` I believe

**Comment by kolyshkin** [comment]
Also, can you please add some performance comparison to commit message?

File: scripts/vps-create.in (hunk 1 above)
Comment (mrqwer88) [inline scripts/vps-create.in]: Yes - fixed it.

**Comment by mrqwer88** [comment]
Add pv output from vzctl create with time and speed to commit message.

**Comment by kolyshkin** [comment]
Sergey, thank you! Merged.

**Pull request merged**